openbb_terminal/alternative/realestate/landRegistry_model.py

```diff
@@ -0,0 +1,266 @@
+"""  UK Land Registry Model """
+__docformat__ = "numpy"
+
+import logging
+import io
+import pandas as pd
+from SPARQLWrapper.Wrapper import CSV, SPARQLWrapper
+
+from openbb_terminal.decorators import log_start_end
+
+logger = logging.getLogger(__name__)
+
+
+def pcd_format(postcode: str):
+    return (
+        postcode[0 : len(postcode) - 3].strip()
+        + " "
+        + postcode[len(postcode) - 3 : len(postcode)]
+    )
+
+
+@log_start_end(log=logger)
+def get_estate_sales(postcode: str, limit: int) -> pd.DataFrame:
```

Comment:
this should provide a default limit. `limit: int = 20` for example,  If not then the sdk requires it to be provided and the UX is not great,